### ISSUE TITLE
Session: Keras model architecture

### ISSUE THREAD

**Issue by gar1t** [gar1t/tensorflow-workshop#8]
This session should probably follow the paper summary (#7). It should cover Keras model basics before moving into the specific architecture we'll be using. This is a code-centric presentation.

The session I think should have some hands-on running of the model, either to train (maybe one epoch) or detect.

At the end the students should feel thoroughly exposed to model development in Keras and know what the Mask RCNN model is about, at least at a high level.
